plugins/rebalance/api/resp.go

```diff
@@ -0,0 +1,70 @@
+package rebalance
+
+import (
+	"github.com/pborman/uuid"
+)
+
+// Status represents Rebalance Status
+type Status uint64
+
+const (
+	// GfDefragStatusNotStarted should be set only for a node in which rebalance process is not started
+	GfDefragStatusNotStarted Status = iota
+	// GfDefragStatusStarted should be set only for a node that has been just started rebalance process
+	GfDefragStatusStarted
+	// GfDefragStatusStopped should be set only for a node that has been just stopped rebalance process
+	GfDefragStatusStopped
+	// GfDefragStatusComplete should be set only for a node that the rebalance process is completed
+	GfDefragStatusComplete
+	// GfDefragStatusFailed should be set only for a node that are failed to run rebalance process
+	GfDefragStatusFailed
+	// GfDefragStatusLayoutFixStarted should be set only for a node that has been just started rebalance fix-layout
+	GfDefragStatusLayoutFixStarted
+	// GfDefragStatusLayoutFixStopped should be set only for a node that has been just stopped rebalance fix-layout
+	GfDefragStatusLayoutFixStopped
+	// GfDefragStatusLayoutFixComplete should be set only for a node that the rebalance fix-layout is completed
+	GfDefragStatusLayoutFixComplete
+	// GfDefragStatusLayoutFixFailed should be set only for a node that are failed to run rebalance fix-layout
+	GfDefragStatusLayoutFixFailed
+)
+
+// Command represents Rebalance Commands
+type Command uint64
+
+const (
```

Comment:
Remove the `GfDefrag` prefix, just use `CmdNone`, `CmdStart` etc.

Make sure the order is same as the Rebalance code of Glusterd1.